notebooks/00_quick_start/sequential_recsys_amazondataset.ipynb

```diff
@@ -0,0 +1,490 @@
+{
```

Comment:
<p>one question, how long does it take to train 10 epochs?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1010/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
I will add a time cost column to the final table in the notebook.